src/databricks/labs/ucx/source_code/jobs.py

```diff
@@ -78,8 +78,8 @@ def as_message(self) -> str:
 
 
 class WorkflowTask(Dependency):
-    def __init__(self, ws: WorkspaceClient, task: jobs.Task, job: jobs.Job):
-        loader = WrappingLoader(WorkflowTaskContainer(ws, task, job))
+    def __init__(self, ws: WorkspaceClient, task: jobs.Task, job: jobs.Job, cache: WorkspaceCache | None = None):
```

Comment:
don't add `| None` as constructor dependencies - it leads to non-deterministic logic and subtle bugs that are harder to diagnose later.